client/src/modules/layouts/Contacts/ContactsService.ts

```diff
@@ -0,0 +1,75 @@
+import ContactData from '../../../models/ContactData';
+import CustomField from '../../../models/customFields/CustomField';
+import CustomFieldSetting from '../../../models/customFields/CustomFieldSetting';
+import { basicColumnStyles, rightAlignStyles } from '../../common/Table/ColumnStyles';
+
+export interface ContactColumn {
+  dataField: string;
+  text: string;
+
+  [name: string]: string | object;
+}
+
+interface Styles {
+  [name: string]: object;
+}
+
+const columnsWithRightAlignment = ['Closed leads', 'Opened leads', 'People'];
```

Comment:
extract string values as constants at least in this file if it doesn't using it other files.